src/tools/tidy/src/main.rs

```diff
@@ -1,8 +1,8 @@
 //! Tidy checks source code in this repository.
 //!
 //! This program runs all of the various tidy checks for style, cleanliness,
-//! etc. This is run by default on `make check` and as part of the auto
-//! builders.
+//! etc. This is run by default on `./x.py test` and as part of the auto
+//! builders. The tidy checks can be executed with `./x.py test src/tools/tidy`.
```

Comment:
There's also a shortcut `./x.py test tidy`.

Reply:
(And `./x.py test toolname` in general.)

Reply:
> There's also a shortcut ./x.py test tidy

Ah! Very nice.  I don't see this in the `./x.py --help` or Rustc guide.  Is this the preferred approach?  Happy to add this to both of those locations and update here if so.

> (And ./x.py test toolname in general.)

Great!

Reply:
>Is this the preferred approach?

Well, it's my preferred approach at least, because it's shorter to type.
I don't know about others.